content/en/technical-overview/architecture/cht-core.md

```diff
@@ -1,36 +1,66 @@
 ---
-title: "Architecture of CHT Instances"
-linkTitle: "Architecture"
-weight: 1
+title: "CHT-Core"
+linkTitle: "CHT-Core"
+weight: 2
 description: >
   Overview of CHT components and interactions
 relatedContent: >  
-  technical-overview/cht-watchdog
-  technical-overview/cht-sync
+  technical-overview/architecture/cht-watchdog
+  technical-overview/architecture/cht-sync
 aliases:
    - /core/overview/architecture
 ---
 
-## Overview
-
-<!-- make updates to this diagram on the google slides:            -->
-<!-- https://docs.google.com/presentation/d/1j4jPsi-gHbiaLBfgYOyru1g_YV98PkBrx2zs7bwhoEQ/ -->
-
-{{< figure src="architecture.png" link="architecture.png" caption="Data Flows" >}}
+{{< hextra/hero-subtitle >}}
+  Technical overview of the CHT Core Framework
+{{< /hextra/hero-subtitle >}}
+
+The CHT Core Framework is the primary component of the CHT. The server comes with authentication, role based authorization, data security, and a range of protected data access endpoints. Read more detail in [cht-core GitHub repository](https://github.com/medic/cht-core).
+
+```mermaid
+flowchart TB
+  subgraph CHT_Core_Framework["<div style="width: 23em; height:2em; display:flex; justify-content: flex-start; align-items:flex-end;">Server</div>"]
+    direction TB
+    NGINX(NGINX):::container
+    API(API):::container
+    Sentinel(Sentinel):::container
+    HAProxy(Haproxy):::container
+    CouchDB[(CouchDB)]:::container
+    classDef container stroke:#63a2c6, fill:#eef5f9, color:#000
+  end
+
+  Upgrade["Upgrade Service"]:::container
+
+  subgraph Client["<div style="width: 28em; height:2em; display:flex; justify-content: flex-start; align-items:flex-end;">Client</div>"]
```

Comment:
Maybe change it from `Client` to `Client (Android & Browser)`?  Don't feel too strongly here